packages/dynamoose/lib/utils/keyBy.ts

```diff
@@ -0,0 +1,5 @@
+import {AnySimpleObject, AnySimpleValue} from "../Types";
+
+// Source: https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore
```

Comment:
Can't do this. Their license is MIT (https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore/blob/master/LICENSE) which conflicts with ours.